package.json

```diff
@@ -10,7 +10,7 @@
     "build": "node script/build.js",
     "preupload": "npm run test",
     "upload": "node script/upload.js",
-    "update": "script/update.sh",
+    "fetch-data-sources": "script/update.sh",
```

Comment:
The workflow has this listed as `update-data-sources` instead of `fetch-data-sources`. I like `update-` better, what do you think? Also, how do you feel about renaming `update.sh` to be `update-data-sources.sh`?

Reply:
renamed e16adda6a33aa6d90a196584a349b51bda37e40e 📋